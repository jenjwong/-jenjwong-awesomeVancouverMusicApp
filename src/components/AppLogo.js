import React from 'react';

const AppLogo = () => {
  return (
    <div className="app-logo-container">
      <div className="app-logo">BANDS NEARBY</div>
      <div className="app-logo-mobile">BN</div>
    </div>
  );
};

export default AppLogo;
