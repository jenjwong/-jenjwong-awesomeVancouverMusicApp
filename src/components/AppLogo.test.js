import React from 'react';
import AppLogo from './AppLogo';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

test('Logo Snapshot test', () => {
  const component = shallow(<AppLogo />);
  const tree = shallowToJson(component);
  expect(tree).toMatchSnapshot();
});
