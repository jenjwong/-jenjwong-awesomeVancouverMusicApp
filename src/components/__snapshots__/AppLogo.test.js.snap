exports[`test Logo Snapshot test 1`] = `
<div
  className="app-logo-container">
  <div
    className="app-logo">
    BANDS NEARBY
  </div>
  <div
    className="app-logo-mobile">
    BN
  </div>
</div>
`;
